examples/tutorials/imagenet_pytorch/model_def.py

```diff
@@ -0,0 +1,159 @@
+                num_workers=self.context.get_hparam("workers", pin_memory=True))
+        
+        elif self.context.get_hparam('dataset') == 'cifar':
+
+            transform = transforms.Compose(
+                [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
+            )
+            trainset = datasets.CIFAR10(
+                root=self.download_directory, train=True, download=True, transform=transform
+            )
+            return DataLoader(trainset, batch_size=self.context.get_per_slot_batch_size())
+
+    def build_validation_data_loader(self):
+        if self.context.get_hparam('dataset') == 'imagenet':
+            valdir = os.path.join(self.download_directory, 'val')
+            self.normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
+                                        std=[0.229, 0.224, 0.225])
+
+            val_dataset = datasets.ImageFolder(
+                traindir,
+                transforms.Compose([
+                    transforms.RandomResizedCrop(224),
+                    transforms.RandomHorizontalFlip(),
+                    transforms.ToTensor(),
+                    self.normalize,
+                ]))
+
+            return DataLoader(
+                val_dataset, batch_size=self.context.get_per_slot_batch_size(), shuffle=False,
+                num_workers=self.context.get_hparam("workers", pin_memory=True))
+        else:
+
+            transform = transforms.Compose(
+                [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
+            )
+            val_dataset = datasets.CIFAR10(
+                root=self.download_directory, train=False, download=True, transform=transform
+            )
+            return DataLoader(val_dataset, batch_size=self.context.get_per_slot_batch_size())
+
+    def train_batch(self, batch: TorchData, epoch_idx: int, batch_idx: int):
+        images, target = batch
+        output = self.model(images)
+        loss = self.criterion(output, target)
+        acc1, acc5 = self.accuracy(output, target, topk=(1, 5))
+
+        self.context.backward(loss)
+        self.context.step_optimizer(self.optimizer)
+
+        return {"loss": loss.item(), 'top1': acc1[0], 'top5': acc5[0]}
+
+    def evaluate_batch(self, batch: TorchData):
+        images, target = batch
+
+        output = self.model(images)
+        loss = self.criterion(output, target)
+
+        # measure accuracy and record loss
+        acc1, acc5 = self.accuracy(output, target, topk=(1, 5))
+
+        return {"val_loss": loss.item(), "top1": acc1[0], "top5":acc5[0]}
+
+    def adjust_learning_rate(self, optimizer, epoch, args):
+        """Sets the learning rate to the initial LR decayed by 10 every 30 epochs"""
+        lr = args.lr * (0.1 ** (epoch // 30))
+        for param_group in self.optimizer.param_groups:
+            param_group['lr'] = lr
```

Comment:
I don't see this being used anywhere. Would it be okay to remove it or perhaps add a comment if it was in the original code?